licenses/LICENSE

```diff
@@ -214,27 +214,32 @@ following licenses.
 ================================================================
 The following component(s) are provided under the Apache-2.0 License (Apache License 2.0).
 See the respective project link for details.
-You can find a copy of the License at LICENSE_go-chassis_foundation.
 
-github.com/go-chassis/foundation (v0.0.0-20190621030543-c3b63f787f4c)
-github.com/go-chassis/go-chassis (v1.7.1-0.20190903133217-e4a22c998fe1)
 github.com/modern-go/concurrent (v0.0.0-20180306012644-bacd9c7ef1dd)
 github.com/modern-go/reflect2 (v1.0.1)
 github.com/prometheus/client_golang (v0.9.1)
 github.com/prometheus/client_model (v0.0.0-20190115171406-56726106282f)
 github.com/prometheus/common (v0.2.0)
 github.com/prometheus/procfs (v0.0.0-20190117184657-bf6a532e95b1)
 go.mongodb.org/mongo-driver (v1.0.3)
-github.com/go-chassis/go-archaius (v0.20.0)
-github.com/go-chassis/go-chassis-config (v0.9.0)
 github.com/matttproud/golang_protobuf_extensions (v1.0.1)
 github.com/mwitkow/go-conntrack (v0.0.0-20161129095857-cc309e4a2223)
 gopkg.in/yaml.v2 (v2.2.1)
-github.com/go-chassis/paas-lager (v1.0.2-0.20190328010332-cf506050ddb2)
 github.com/xdg/scram (v0.0.0-20180814205039-7eeb5667e42c)
 github.com/xdg/stringprep (v1.0.0)
 github.com/go-mesh/openlogging (v1.0.1-0.20181205082104-3d418c478b2d)
 
+================================================================
+The following component(s) are provided under the Apache-2.0 License (Apache License 2.0).
+See the respective project link for details.
+You can find a copy of the License at LICENSE_go-chassis_go-chassis.
```

Comment:
Could you rename the Licencese file name to LICENSE_Apache2.0 instead of using LICENSE_go-chassis_go-chassis.

Reply:
changed